daal4py/sklearn/monkeypatch/dispatcher.py

```diff
@@ -145,6 +146,16 @@ def enable(name=None, verbose=True):
     else:
         for key in _get_map_of_algorithms():
             do_patch(key)
+    if deprecation:
+        warn("Scikit-learn patching with daal4py is deprecated "
+            "and will be removed in the future.\n"
+            "Please, use Intel(R) Extension for Scikit-learn module instead "
+            "(pip install intel-sklearn-extension)\n"
+            "To enable patching, please, use one of options:\n"
+            "1) python -m iskex <your_script>\n"
+            "2) from iskex import patch_sklearn\n"
+            "   patch_sklearn()", 
+             FutureWarning, stacklevel=2)
```

Comment:
```DeprecationWarning``` probably?

Reply:
DeprecationWarning are intended for other Python developers, and FutureWarning for users
https://docs.python.org/3/library/warnings.html
For example, scikit-learn using FutureWarning
https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/cluster/_kmeans.py#L787